src/main/java/teammates/common/datatransfer/attributes/NotificationAttributes.java

```diff
@@ -179,7 +191,7 @@ public void setUpdatedAt(Instant updatedAt) {
     @Override
     public Notification toEntity() {
         return new Notification(notificationId, startTime, endTime, type,
-                targetUser, title, message, shown, createdAt, updatedAt);
+                targetUser, title, message, shown, updatedAt);
```

Comment:
I think this is not necessary, see my note in `Notification.java`

Reply:
After discussion, we decide to add a conditional check here so that createdAt could be set to `Instant.now()` upon initialization.